api/origin_trials_api.py

```diff
@@ -34,8 +34,8 @@ def do_get(self, **kwargs):
           f'{settings.OT_API_URL}/v1/trials',
           params={'prettyPrint': 'false', 'key': key})
       response.raise_for_status()
-      trials = response.json()['trials']
-      return trials
-    except:
+    except requests.exceptions.HTTPError:
       self.abort(500, 'Error obtaining origin trial data from API.')
```

Comment:
You should print a log with an exception level as well. (So that we get an email alert of some sorts)

Reply:
I went to go do this, but it seems logging is already implemented in the `self.abort` call

https://github.com/GoogleChrome/chromium-dashboard/blob/main/framework/basehandlers.py#L67-L71

Reply:
Oh ok! Thanks for pointing that out.